mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -633,6 +641,29 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
                     &recreate_ctx.taker_coin,
                 )?,
             }),
+            TakerSwapEvent::TakerPaymentSentAndPreimageSendingSkipped {
+                maker_coin_start_block,
+                taker_coin_start_block,
+                taker_payment,
+                maker_payment,
+                negotiation_data,
+            } => Box::new(TakerPaymentSent {
```

Comment:
Why make a new struct in maker swap but reuse the TakerPaymentSent Struct here?

Reply:
ooooooh, thanks for the catch! should be TakerPaymentSentAndPreimageSendingSkipped

Reply:
fixed https://github.com/KomodoPlatform/komodo-defi-framework/pull/2359/commits/2f23f7041470eeeca8a347af4757975656712b08